api/users/views.py

```diff
@@ -603,6 +611,63 @@ def create(self, request, *args, **kwargs):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
 
+class UserChangePassword(JSONAPIBaseView, generics.CreateAPIView, UserMixin):
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+        CurrentUser,
+    )
+
+    required_read_scopes = [CoreScopes.NULL]
+    required_write_scopes = [CoreScopes.USER_SETTINGS_WRITE]
+
+    view_category = 'users'
+    view_name = 'user_password'
+
+    serializer_class = UserChangePasswordSerializer
+    throttle_classes = (ChangePasswordThrottle, )
```

Comment:
Can you explain why this throttle class is necessary?

Reply:
So I'm not 100%, but I thought the preexisting behavior was to throttle correct password attempts (that's above) and incorrect ones where you get 3 invalid attempts.